### PR TITLE
refactor: Upgrade to Prettier 3 (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "nock": "^13.3.2",
     "nodemon": "^3.0.1",
     "p-limit": "^3.1.0",
-    "prettier": "^2.8.8",
+    "prettier": "^3.0.0",
```
Comment (netroy) [inline package.json:55]: we are still pulling in older versions of prettier as transient dependencies. maybe we should wait until all other dependencies have been updated to use prettier 3.x?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,1395 +1,1200 @@
...
-* **editor:** Add "Download" button if JSON data is to large ([#6850](https://github.com/n8n-io/n8n/issues/6850)) ([efe08cc](https://github.com/n8n-io/n8n/commit/efe08cced3979e40bbf011301f51f9f2f69117de))
-* **editor:** Migrate Design System and Editor UI to Vue 3 ([#6476](https://github.com/n8n-io/n8n/issues/6476)) ([dd6a4c9](https://github.com/n8n-io/n8n/commit/dd6a4c956adb997b59ee0174eedd9b8f6265ec6b)), closes [#6571](https://github.com/n8n-io/n8n/issues/6571)
-* **Facebook Graph API Node:** Add support for v16 and v17 ([#6808](https://github.com/n8n-io/n8n/issues/6808)) ([46a41c1](https://github.com/n8n-io/n8n/commit/46a41c1c91c7ebadd72fb51b6f749a542a7393fc))
-* **Pipedrive Node:** Add option to update the file name and description ([#6883](https://github.com/n8n-io/n8n/issues/6883)) ([f8ad543](https://github.com/n8n-io/n8n/commit/f8ad543af5c5d680d23d5dab861a6b3fe47b4ccc))
-
+- Auth.api user limit test expecting incorrect status ([#6836](https://github.com/n8n-io/n8n/issues/6836)) ([371bfa0](https://github.com/n8n-io/n8n/commit/371bfa0f488f0a240bd1a8854759d6f6639e6eea))
```
Comment (netroy) [inline CHANGELOG.md:5]: This file is autogenerated, and should be added to `.prettierignore` to avoid conflicts during the release process.

**Comment by ivov** [comment]
Will reopen later without changes to tests.

File: package.json (hunk 1 above)
Comment (ivov) [reply package.json:55]: From I can tell, the older versions of Prettier are coming from `@storybook/cli` and `@storybook/codemod`

https://github.com/storybookjs/storybook/blob/377f7f9c0a022ae073ab0e93792d4e15dd88b15c/code/lib/cli/package.json#L87

https://github.com/storybookjs/storybook/blob/377f7f9c0a022ae073ab0e93792d4e15dd88b15c/code/lib/codemod/package.json#L60

File: CHANGELOG.md (hunk 2 above)
Comment (ivov) [reply CHANGELOG.md:5]: Will remove.

**Comment by ivov** [comment]
@netroy I was planning to reopen without changes to tests, to avoid having to debug the diffs, esp. on workflow JSONs in nodes tests. Should we not reopen then based on the two deps that still have Prettier at 2.8.8?

**Pull request closed without merging**